src/main/java/org/mitre/synthea/export/BFDExportBuilder.java

```diff
@@ -1,159 +1,483 @@
+              entry.getLineNum(), type, entry.getField(), expression);
+        retval = false;
+      } else if (expression.startsWith("[")) { // either a function or developer note
+        retval = true;
+      } else {
+        retval = true;
+      }
+    } else {
+      // System.out.println("rejecting because expression="+expression);
+      retval = false;
+    }
+    return retval;
+  }
+
+  private String evalConfig(
+        String expression, 
+        BFDExportConfigEntry entry, 
+        ExportConfigType type, 
+        HealthRecord.Encounter encounter,
+        HealthRecord.Device device,
+        Person person) {
+    String retval = expression;
+    // look for comments (matching anything in parenthesis)
+    int commentStart = expression.indexOf("(");
+    if (commentStart >= 0) {
+      retval = expression.substring(0, commentStart - 1); // peel off comment
+      // currently we just ignore comments 
+      //  since they are for the analyst doing the config spreadsheet
+      // comment = expression.substring(commentStart + 1, expression.length()-1);
+    }
+    boolean printError = false;
+
+    // evaluate for functions
+    if (expression.startsWith("[")) {
+      switch (expression) {
+        case "[Blank]":
+        case "[blank]":
+          retval = "";
+          break;
+        case "[bene_id]":
+          retval = (String) person.attributes.get(BB2_BENE_ID);
+          break;
+        case "[bene_race]":
+          retval = BB2RIFExporter.bb2RaceCode(
+            (String)person.attributes.get(Person.ETHNICITY),
+            (String)person.attributes.get(Person.RACE));
+          break;
+        case "[bene_stateCode]":
+          if (locationMapper != null) {
+            retval = locationMapper.getStateCode((String)person.attributes.get(Person.STATE));
+          } else {
+            retval = "?";
+          }
+          break;
+        case "[bene_terminationCode]":
+          retval = (person.attributes.get(Person.DEATHDATE) == null) ? "0" : "1";
+          break;
+        case "[carr_num_from_state]":
+          retval = lookupCarrier(encounter.provider.state, type, entry.getField());
+          break;
+        case "[device_startTimestamp]":
+          retval = bb2DateFromTimestamp(device.start);
+          break;
+        case "[encounter_claimCoinsurancePaid]":
+          retval = String.format("%.2f", encounter.claim.getCoinsurancePaid());
+          break;
+        case "[encounter_claimDeductiblePaid]":
+          retval = String.format("%.2f", encounter.claim.getDeductiblePaid());
+          break;
+        case "[encounter_claimDeductible+Coinsurance]":
+          retval = String.format("%.2f", 
+                encounter.claim.getDeductiblePaid() + encounter.claim.getCoinsurancePaid());
+          break;
+        case "[encounter_claimTotalCost]":
+          retval = String.format("%.2f", encounter.claim.getTotalClaimCost());
+          break;
```

Comment:
I do not want this type of custom logic abstracted away in a domain-specific language buried in a TSV configuration file.

Don't change it though. Leave it as is.